observability/logging/logging-6.2/log6x-clf-6.2.adoc

```diff
@@ -0,0 +1,122 @@
+
+[id="modifying-log-level_6-2_{context}"]
+== Modifying log level in collector
+
+To modify the log level in the collector, you can set the `observability.openshift.io/log-level` annotation to `trace`, `debug`, `info`, `warn`, `error`, and `off`.
+
+.Example log level annotation
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  annotations:
+    observability.openshift.io/log-level: debug
+# ...
+----
+
+[id="managing-the-operator_6-2_{context}"]
+== Managing the Operator
+
+The `ClusterLogForwarder` resource has a `managementState` field that controls whether the operator actively manages its resources or leaves them Unmanaged:
+
+Managed:: (default) The operator will drive the logging resources to match the desired state in the CLF spec.
+
+Unmanaged:: The operator will not take any action related to the logging components.
+
+This allows administrators to temporarily pause log forwarding by setting `managementState` to `Unmanaged`.
+
+[id="clf-structure_6-2_{context}"]
+== Structure of the ClusterLogForwarder
+
+The CLF has a `spec` section that contains the following key components:
+
+Inputs:: Select log messages to be forwarded. Built-in input types `application`, `infrastructure` and `audit` forward logs from different parts of the cluster. You can also define custom inputs.
+
+Outputs:: Define destinations to forward logs to. Each output has a unique name and type-specific configuration.
+
+Pipelines:: Define the path logs take from inputs, through filters, to outputs. Pipelines have a unique name and consist of a list of input, output and filter names.
+
+Filters:: Transform or drop log messages in the pipeline. Users can define filters that match certain log fields and drop or modify the messages. Filters are applied in the order specified in the pipeline.
+
+[id="clf-inputs_6-2_{context}"]
+=== Inputs
+
+Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
+
+application:: Selects logs from all application containers, excluding those in infrastructure namespaces.
+
+infrastructure:: Selects logs from nodes and from infrastructure components running in the following namespaces:
+** `default`
+** `kube`
+** `openshift`   
+** Containing the `kube-` or `openshift-` prefix
+
+audit:: Selects logs from the OpenShift API server audit logs, Kubernetes API server audit logs, ovn audit logs, and node audit logs from auditd.
+
+Users can define custom inputs of type `application` that select logs from specific namespaces or using pod labels.
+
+[id="clf-outputs_6-2_{context}"]
+=== Outputs
+
+Outputs are configured in an array under `spec.outputs`. Each output must have a unique name and a type. Supported types are:
+
+azureMonitor:: Forwards logs to Azure Monitor.
+cloudwatch:: Forwards logs to AWS CloudWatch.
+//elasticsearch:: Forwards logs to an external Elasticsearch instance.
```

Comment:
we should include elasticsearch. It is still supported in 6.x

Reply:
I thought Elasticsearch was not supported in 6.x, @JoaoBraveCoding or @jcantrill can you please confirm?

Reply:
@anpingli is correct. In 6.x we no longer support elastic as a log store however the CLO still offers the ability for users to forward their logs to an external elasticsearch instance.

Reply:
Thanks, @JoaoBraveCoding for the explanation, added Elasticsearch back. cc @anpingli